cumulusci/robotframework/utils.py

```diff
@@ -229,3 +232,31 @@ def wrapper(*args, **kwargs):
             raise
 
     return wrapper
+
+
+def get_locator_module_name(version):
+    """Return module name of locator file for the specified version
+
+    If a file for the specified version can't be found, the locator
+    file with the highest number will be returned. Passing in None
+    effectively means "give me the latest version".
+    """
+
+    here = os.path.dirname(__file__)
+    if os.path.exists(os.path.join(here, f"locators_{version}.py")):
+        # our work here is done.
+        locator_filename = f"locators_{version}.py"
+    else:
+
+        def by_num(filename):
+            """Pull out the number from a filename"""
+            nums = re.findall(r"_(\d+)", filename)
+            if nums:
+                return int(nums[-1])
+            return 0
+
+        files = sorted(glob.glob1(here, "locators_*.py"), key=by_num)
+        locator_filename = files[-1]
+
+    locator_module_name = f"cumulusci.robotframework.{locator_filename[:-3]}"
```

Comment:
Should we log a warning-level message here that we're using a fallback locator library, so it'll be clear in the logs?

Reply:
Thank you very much for this comment! I started to add it earlier, but logging with robot doesn't always play nice with pytest. However, this forced me to take a step back and look at the problem with fresh eyes, and was reminded that robot has a little-used logging API that handles the case of logging outside of the context of a running robot test. I'm glad I was forced to think a bit harder about this problem.